src/config.rs

```diff
@@ -401,6 +355,148 @@ impl DbConfig {
     }
 }
 
+cf_config!(RaftDefaultCfConfig);
+
+impl Default for RaftDefaultCfConfig {
+    fn default() -> RaftDefaultCfConfig {
+        RaftDefaultCfConfig {
+            block_size: ReadableSize::kb(64),
+            block_cache_size: ReadableSize::mb(memory_mb_for_cf(true, CF_DEFAULT) as u64),
+            cache_index_and_filter_blocks: true,
+            use_bloom_filter: false,
+            whole_key_filtering: true,
+            bloom_filter_bits_per_key: 10,
+            block_based_bloom_filter: false,
+            compression_per_level: [DBCompressionType::No,
+                                    DBCompressionType::No,
+                                    DBCompressionType::Lz4,
+                                    DBCompressionType::Lz4,
+                                    DBCompressionType::Lz4,
+                                    DBCompressionType::Zstd,
+                                    DBCompressionType::Zstd],
+            write_buffer_size: ReadableSize::mb(128),
+            max_write_buffer_number: 5,
+            min_write_buffer_number_to_merge: 1,
+            max_bytes_for_level_base: ReadableSize::mb(512),
+            target_file_size_base: ReadableSize::mb(32),
+            level0_file_num_compaction_trigger: 4,
+            level0_slowdown_writes_trigger: 20,
+            level0_stop_writes_trigger: 36,
+            max_compaction_bytes: ReadableSize::gb(2),
+            compaction_pri: CompactionPriority::ByCompensatedSize,
+        }
+    }
+}
+
+impl RaftDefaultCfConfig {
+    pub fn build_opt(&self) -> ColumnFamilyOptions {
+        let mut cf_opts = build_cf_opt!(self);
+        let f = Box::new(FixedPrefixSliceTransform::new(region_raft_prefix_len()));
+        cf_opts.set_memtable_insert_hint_prefix_extractor("RaftPrefixSliceTransform", f)
+            .unwrap();
+        cf_opts
+    }
+}
+
+#[derive(Clone, Serialize, Deserialize)]
+#[serde(default)]
+#[serde(rename_all = "kebab-case")]
+pub struct RaftDbConfig {
```

Comment:
yeah, only use DbConfig is hard to distinguish kv and raft db, because TiKvConfig::default has not parameter.
maybe we can use two xxDbConfig with other common config DbConfig